### DISCUSSION TITLE
如何自定义一套快捷键？

### DISCUSSION THREAD

**Discussion by Wind-stormger** [diana7127/mpv.net-DW#17] [Q&A] [answered]
我持续使用[mpv.net-DW](https://github.com/diana7127/mpv.net-DW)一个月了，发现我每次都是固定做三个操作，开启A4K, 补帧，设置帧率为60hz，请问如何自定义一套快捷键，一键实现三个操作？

**Comment by diana7127** [comment]
看下readme的[使用说明](https://github.com/diana7127/mpv.net-DW#%E8%AE%BE%E7%BD%AE%E9%80%89%E9%A1%B9-1)哦

**Comment by Wind-stormger** [comment]
![image](https://github.com/diana7127/mpv.net-DW/assets/76462385/660f4bd5-8d1f-4a8a-9531-bc6027e2e6e2)
![image](https://github.com/diana7127/mpv.net-DW/assets/76462385/1e3cf05b-aa37-4edf-a034-0ae1db0a1b3d)

格式应该是正确的，却并没有起作用？

**Comment by Wind-stormger** [comment]
![image](https://github.com/diana7127/mpv.net-DW/assets/76462385/14453da1-4d99-4ebb-ad36-a4b36c4fd42e)
@diana7127

**Comment by diana7127** [comment, marked as answer]
mpv_lazy的vs滤镜改版之后，不能用之前的写法了，vf的选项框填入`vapoursynth=~~/vs/MEMC_MVT_LQ.vpy`

**Reply by Wind-stormger** [reply]
舒服了，感谢。
